Scripts/Core/EditorVR.UI.cs

```diff
@@ -15,7 +15,7 @@ partial class EditorVR
 
 		class UI : Nested, IInterfaceConnector
 		{
-			const byte k_MinStencilRef = 5;
+			const byte k_MinStencilRef = 2;
```

Comment:
The stencil from the outline shader caused the UI mask on the menu to bleed through. The fix ended up resolving the need for different stencil refs per outline material. Thus, refs 2, 3, and 4 go back to the pool.